sheeping/sheep_localizer.py

```diff
@@ -88,12 +88,13 @@ def visualize_results(self, image, bboxes, scores, scaling=(1, 1)):
 
             # scale bounding box with scale factor
             bbox = [bbox[0] * scaling[1], bbox[1] * scaling[0], bbox[2] * scaling[1], bbox[3] * scaling[0]]
-            bbox = list(map(int, map(round, bbox)))
+            bbox = list(map(lambda x: int(round(x)), bbox))
 
             width = bbox[3] - bbox[1]
             height = bbox[2] - bbox[0]
 
-            cv2.rectangle(image, (bbox[1], bbox[0]), (bbox[1] + width, bbox[0] + height), self.color)
+            thickness = 1 + max(image.shape) // 500
```

Comment:
why do you divide by `500`? What happens if we use SSD300?

Reply:
Because the width of the prediction rectangle should scale with image width/height. It does not depend on the model (the rectangle is drawn on the original image (e.g. with size 3264x1836)).